app/src/main/java/project/java/tbusdriver/Controller/Fragments/Auth.java

```diff
@@ -182,11 +196,23 @@ protected void onProgressUpdate(String... values) {
             else {
                 EditText authCode = (EditText) myActivity.findViewById(R.id.authCode);
                 authCode.setText("");
-                if (checkBoxIsCheck == true) {
+                if (checkBoxIsCheck == true)
+                {
```

Comment:
in java, and mostly at android you put the { at the same line with the statement.
for exsmple: while (true) {
it's because there is a lot of anonymous class at java